### ISSUE TITLE
Would an 'anchorsSelector' be possible

### ISSUE THREAD

**Issue by djowett** [alvarotrigo/fullPage.js#1380]
.... as an alternative to listing anchors?

**Comment by alvarotrigo** [comment]
You mean... using a html attribute instead of the array in the plugin options?

**Comment by djowett** [comment]
I mean, rather than providing this:

    $('body.template-fullpage_view #content').fullpage({{
        ...
        anchors: ['home', 'our-products', 'media']
        menu: '.navTreeLevel0',
    }});

(and having to calculate 'home', 'our-products' etc...)
I can simply do this:

    $('body.template-fullpage_view #content').fullpage({{
        ...
        anchorsSelector: '.navTreeLevel0 > .navTreeItem',
        menu: '.navTreeLevel0',
    }});

**Comment by alvarotrigo** [comment]
I believe another better approach would be to just set it directly in the HTML markup:

```html
<section data-anchor="firstPage"></section>
<section data-anchor="secondPage"></section>
```

**Comment by djowett** [comment]
Sure, that would be consistent with the was slides are handled. I'm just as happy with that approach.

**Comment by alvarotrigo** [comment]
I'll take a look at it for the next release.

**Comment by djowett** [comment]
Thanks.  In case it helps anyone in the meantime, I have calculated the anchor parameter for my fullpage call as follows.  Obviously, this works for my framework, will need tweaking for other setups.... 

    anchorList = [];
    classFinder = /section-([^\s]*)/;

    $('.navTreeLevel0 > .navTreeItem').each(function() {
        sectionClassMatch = $(this).attr('class').match(classFinder);
        if (sectionClassMatch !== null) {
            anchorList.push(sectionClassMatch[1]);
        }
        else {
            anchorList.push('NotFound');
        }
    });

    $('body.template-fullpage_view #content').fullpage({
        sectionSelector: '#content > div.section',
        slideSelector: '#content div.slide',
        verticalCentered: false,
        anchors: anchorList,
        menu: '.navTreeLevel0',
    });

**Comment by alvarotrigo** [comment]
Implemented in 2.6.8.
